### ISSUE TITLE
"brace expansion is not supported" false positive

### ISSUE THREAD

**Issue by SpecLad** [koalaman/shellcheck#481]
For this code:

```sh
#!/bin/sh
find -exec echo {} ';'
```

ShellCheck says "SC2039 In POSIX sh, brace expansion is not supported.". But `{}` is not affected by brace expansion, since there are no commas in it. So I think this is a false positive.

**Comment by Arthur2e5** [comment]
So does `echo {a}`. In bash, brace expansion is performed only if `,` is found, to afford 'the greatest degree of sh compatibility.' ([bach doc/FAQ](http://git.savannah.gnu.org/cgit/bash.git/tree/doc/FAQ#n1432))

**Comment by koalaman** [comment]
You are both correct, well spotted! This has been fixed in 5569292. 

The bash faq does not consider `{a..z}` which expands even without an unquoted comma, but this patch does.
